app/plugins/Overview/js/overview.js

```diff
@@ -81,6 +80,8 @@ IPC.on('wallet-update', function(err, result) {
 	} else {
 		updateField(err, 'Locked', '', 'lock');
 	}
+
+	updateField(err, 'Balance: ', unlocked ? bal : '---', 'balance');
```

Comment:
The balance field is dependent upon the unlock state so it had to be moved below it, I just brought the `bal` initialization down next to it so they're together.

`result === null` adjustment has also been made.